plaid/api.go

```diff
@@ -0,0 +1,31 @@
+package plaid
+
+import "os"
+
+// APIResponse is the base struct for all responses from the Plaid API.
+type APIResponse struct {
+	RequestID string `json:"request_id"`
+}
+
+// secretAPIRequest is used to make private requests with a secret key.
+type secretAPIRequest struct {
+	ClientID    string `json:"client_id"`
+	Secret      string `json:"secret"`
+	AccessToken string `json:"access_token"`
+}
+
+// Credentials and parameters to be used for testing.
```

Comment:
i agree with not exporting these variables since they should be only used for testing. also, the developer can already run `main.go` independently without these variables.

Reply:
👍 sounds good.